### DISCUSSION TITLE
Okd openshift support

### DISCUSSION THREAD

**Discussion by yoramda** [ThreeMammals/Ocelot#1894] [Q&A]
## Expected Behavior / New Feature
service discovery support for okd (open shift)

## Actual Behavior / Motivation for New Feature
we are going to upgrade from k8s vanilla to okd (openshift), i would like to know if it is supported ?

**Comment by raman-m** [comment]
Hi @yoramda !
Are you still with Ocelot?

------
> i would like to know if OpenShift is supported ?

**No!** Officially we try to support K8s but not OpenShift.
If you have an intention to contribute we can discuss how to move in this direction.

**Reply by raman-m** [reply]
@ggnaegi FYI
Seems we have limited support of K8s.
I don't remember any issues we've worked on K8s package during 6 months... Unfortunately...
